tempesta_fw/sock_srv.c

```diff
@@ -471,6 +438,10 @@ tfw_sock_srv_start(void)
 static void
 tfw_sock_srv_stop(void)
 {
+	/*
+	 * Connections list is read-only at run time for now, so no need
+	 * to synchronize the list access or disable softirqs.
+	 */
```

Comment:
Don't we need to wait until the connections are closed by Tempesta (which is different from having the sockets closed).